src/components/formElements/InputField.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { describe } from "vitest";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import InputField from "./InputField.tsx";
+import { ThemeProvider } from "styled-components";
+import { theme } from "../../theme/theme.ts";
+
+describe("Input field attributes are accepted", () => {
+  it("renders placeholder text", () => {
+    render(
+      <ThemeProvider theme={theme}>
+        <InputField placeholder={"placeholder text"} />
+      </ThemeProvider>,
+    );
+
+    const inputElement: HTMLInputElement =
```

Comment:
hm, that type should not be necessary

Reply:
I added it to get rid of TS error below:

![Screenshot 2024-04-29 at 16 07 42](https://github.com/gatinhap/plant-app-ver_0/assets/72195350/e69a7a3f-19da-4ac9-9e4c-b11173adccd7).

However, while it's added like this: `const inputElement: HTMLInputElement = screen.getByTestId("input");`

I get another TS error😅:

![Screenshot 2024-04-29 at 16 10 13](https://github.com/gatinhap/plant-app-ver_0/assets/72195350/795afed7-80f3-4e41-87d8-3becbfa9fcbb)

Do you have any suggestion how I could solve it?

Reply:
inputElement.files?[0] should do this. TS is right - there's no certainty that files were uploaded.